tools/ci-operator-prowgen/main.go

```diff
@@ -0,0 +1,126 @@
+package main
+
+import (
```

Comment:
nit: we group imports with stdlib/external/internal like

```go
import (
	"encoding/json"
	"flag"
	"fmt"
	"io/ioutil"
	"os"
	"path"
	"strings"
	
	"github.com/ghodss/yaml"
	cioperatorapi "github.com/openshift/ci-operator/pkg/api"
	prowconfig "k8s.io/test-infra/prow/config"
	prowkube "k8s.io/test-infra/prow/kube"
)
```

Reply:
Aha, blank line makes the formatter do what I want!